README.md

```diff
@@ -272,7 +272,23 @@ should mention how to instantiate those variables/files, etc.
 
 ### Testing
 
-(TODO: Write me)
+We believe testing is an important part of producing high quality
+software. Specifically, we believe robust testing helps improve
+maintaiability as well as reliability of the produced software.
+
+Some general recommendations are:
+* Spend time implement testing.
```

Comment:
"Spend time implement testing" sounds a bit strange, maybe change to "Implement testing code as much as possible"

Reply:
I'm fairly convinced there's almost a point where the invested energy does not result in improvements that are worth the investment, and this may well come long before total test coverage.

The intention here is rather to exhort people to spend some time with testing.

Reply:
I agree that the grammar is weird though - "Spend time *to* implement testing" or "Spend time implementing testing" would work!

Reply:
Also in the highlighted text: `maintaiability` -> `maintainability`